specs/core/0_beacon-chain.md

```diff
@@ -556,17 +557,6 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 }
 ```
 
-#### `DepositRootVote`
```

Comment:
- `DepositRootVote` is still being used in https://github.com/ethereum/eth2.0-specs/blob/212062fc4e7d8f7cd0857ceb42a3a2fd75f16d60/specs/core/0_beacon-chain.md#deposit-root
- The whole "Per-block processing - Deposit Root" section needs to be updated to "PoW Chain Root" version.
- Do we need to update `BeaconBlock.deposit_root: Hash32` -> `BeaconBlock.pow_chain_data: PoWChainData` so that the block proposers are voting on the PoW chain block hash?

Reply:
All done!